infrastructure-playbooks/purge-container-cluster.yml

```diff
@@ -645,6 +645,23 @@
         ansible_os_family == 'RedHat' and
         not is_atomic
 
+    - name: find any service-cid file left
+      find:
+        paths: /run
+        patterns: "ceph-*.service-cid"
+      register: service_cid_files
+
+    - name: rm any service-cid file
+      file:
+        path: "{{ item.path }}"
+        state: absent
+      with_items: "{{ service_cid_files.files }}"
```

Comment:
I think we should manage this in the systemd unit template rather only in the purge.

IMHO the file should be removed when the service is stop. Or maybe have both solutions.

What do you think ?

Reply:
we already remove them in out systemd templates, see https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mon/templates/ceph-mon.service.j2#L13